.vscode/settings.json

```
@@ -6,5 +6,8 @@
 			],
 			"url": "/can_bus/json/schema_tx.json"
 		}
-	]
+	],
+	"cmake.buildDirectory": "${workspaceFolder}/build_${env:PLATFORM}",
+	"cmake.configureOnEdit": false,
+	"cmake.configureOnOpen": true
 }
```

nit: missing newline
```suggestion
}

```